config/kubernetes-sigs/sig-api-machinery/teams.yaml

```diff
@@ -59,27 +59,15 @@ teams:
   controller-tools-admins:
     description: admin access to controller-tools
     members:
-    - DirectXMan12
-    - droot
-    - pwittrock
-    - seans3
-    privacy: closed
-  controller-tools-contributors:
-    description: read access to controller-tools
-    members:
-    - Birdrock
-    - pwittrock
+    - joelanford
+    - vincepri
     privacy: closed
   controller-tools-maintainers:
     description: write access to controller-tools
     members:
     - alvaroaleman
-    - camilamacedo86
```

Comment:
@camilamacedo86 still looks actively involved in this project.  Please restore her.

Reply:
While active, this isn't in line with the https://github.com/kubernetes-sigs/controller-tools/blob/master/OWNERS_ALIASES and its history; this PR just sync the actual group permissions with what's reflected in the repository.

Reply:
It looks like a similar paperwork maneuver was used to update that file recently too: https://github.com/kubernetes-sigs/controller-tools/pull/799

Who are the people who actually maintain controller-tools?  A dev-stats screenshot for controller-tools perhaps?

Reply:
Going by the metrics [here](https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?orgId=1&var-period_name=Last%203%20years&var-metric=prs&var-repogroup_name=All&var-repo_name=kubernetes-sigs/controller-tools&var-country_name=All)

PR Created: 
<img width="1456" alt="CleanShot 2023-05-18 at 11 46 08@2x" src="https://github.com/kubernetes/org/assets/3118335/dacde003-895c-4cd4-acc1-deb42c724a69">

PR Reviewed:
<img width="1444" alt="CleanShot 2023-05-18 at 11 46 52@2x" src="https://github.com/kubernetes/org/assets/3118335/b230f8de-eceb-4732-ab20-b1dd479dde5d">

PR Approved:
<img width="1468" alt="CleanShot 2023-05-18 at 11 47 16@2x" src="https://github.com/kubernetes/org/assets/3118335/6b44eb30-f52c-439a-9d4d-f1e4e5e55a73">

Reply:
Hi @deads2k, 

I am not in the Owner alias, and I am totally fine about removing myself from here. 

My primary focus is on Kubebuilder. Unfortunately, I do not have the bandwidth to contribute as much as I’d like to all projects. These permissions were initially granted by Solly and others due to the significant dependence of Kubebuilder on these projects, when Solly step aside. I never have been in the Owner alias. Those were added to add intended to enable us to manage emergency releases or fast-track specific issues as required swiftly.

@vincepri @joelanford  has been doing the releases, etc. So, it is totally fine. 

Btw, thank you for all consideration

Reply:
Hi @deads2k, 

I am not in the Owner alias, and I am totally fine OK with removing myself from here. 

My primary focus is on Kubebuilder. Unfortunately, I do not have the bandwidth to contribute as much as I’d like to all projects.Btw, thank you for all your consideration
 
@vincepri and @joelanford have been taking care of the project so I think that is right.

Reply:
> Hi @deads2k,
> 
> I am not in the Owner alias, and I am totally fine OK with removing myself from here.

Ok.  According to Vince's devstats above, you are the top code reviewer for the repo, but if it's not something you have the bandwidth that's understandable.

Reply:
Absolutely, @camilamacedo86 contributions in controller-tools have been more than invaluable for the project; the purpose of updating this list is mostly to keep the write permissions to the repository itself scoped to the smallest subset of folks that need it, mostly to push tags or do releases in GitHub.

We can always add a `controller-tools-reviewers` group like other projects have (incl. controller-runtime), which we haven't done yet as nobody stepped up and didn't come up before, how does that sound?